src/appsco-directory-page.html

```diff
@@ -779,6 +779,7 @@
             },
 
             filterAccountsByTerm: function(term) {
```

Comment:
This method should be called different now since it is not only accounts search. Maybe 'filterDirectoryResourcesByTerm'. And this method is called from appsco-app.html so its call should be changed as well.